### PR TITLE
Add community ticket

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: docs/conf/portland/2018/tickets.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Tickets
 =======
 
-**Tickets on sale now!** Follow the links below for each ticket type. 
+**Tickets on sale now!**
```
Comment (ericholscher) [inline docs/conf/portland/2018/tickets.rst:7]: I'd put a button or call out of some kind here, so people who want can just click through.

**Review by ericholscher — APPROVED** [review]

File: docs/conf/portland/2018/tickets.rst (hunk 1 above)
Comment (ericholscher) [reply docs/conf/portland/2018/tickets.rst:7]: I believe we should already have some defined in the HTML/CSS somewhere. Can probably just steal it from the index, unless the CSS styles are super targeted.

Comment (plaindocs) [reply docs/conf/portland/2018/tickets.rst:7]: cta

Looks kinda meh if you just throw it straight in

Comment (plaindocs) [reply docs/conf/portland/2018/tickets.rst:7]: I guess I could live with it floated right and vertical spacing sorted. 

Thoughts?

**Comment by plaindocs** [comment]
There is also this https://ti.to/docs/widget

**Comment by ericholscher** [comment]
Yea, that styling is not great :) I say just ship it, probably not a big deal as we'll sell out anyway :D

**Comment by ericholscher** [comment]
We should also confirm the sponsorship address. I think it's `sponsorship@`, but I _believe_ both should work. Probably just just have it be `sponsor@`, though people might still send to `sponsors@`

**Pull request merged**